fastlane/lib/fastlane/swift_fastlane_function.rb

```diff
@@ -116,6 +116,7 @@ def override_default_value_if_not_correct_type(param_name: nil, param_type: nil,
     end
 
     def get_type(param: nil, default_value: nil, optional: nil, param_type_override: nil, is_string: true)
+      require 'bigdecimal'
```

Comment:
Do we need this one over here?

Reply:
I _think_ we still want this here just for safety. `require bigdecimal` is required to use `BigDecimal`. I tested this out in `irb`. What is most likely happening is another library/dependency is also requiring `bigdecimal` which is why it works but we don’t _really_ want to rely on that 🤷‍♂️